README.md

```diff
@@ -29,10 +30,10 @@ O OctoPost foi projetado para simplificar o processo de compartilhamento de cont
 
 Para começar com o OctoPost, siga estes passos:

```

Comment:
Aqui seria bom adicionar um TO-DOzinho de quais programas você precisa para iniciar o projeto. Por exemplo

\### Instale o Git
Link para o site do git
\### Instale o pnpm
Link para o site do pnpm
(Aqui é importante mencionar que a instalação do pnpm pelo `standalone script` é mais rápido que instalar pelo npm install -g pnpm)